gtsam/navigation/tests/testNavState.cpp

```diff
@@ -192,6 +192,49 @@ TEST(NavState, Coriolis2) {
   EXPECT(assert_equal(numericalDerivative21(coriolis, state2, true), aH));
 }
 
+TEST(NavState, Coriolis3) {
```

Comment:
I added this basic unit test. It fails on the old NavState, and passes on my branch.

Reply:
Nice!